### PR TITLE
Update anchor_generator.py to fix docstring

### PR DESCRIPTION
**Pull request by LukeWood**
documentation fix

### CODE REVIEW

File: keras_cv/layers/object_detection/anchor_generator.py [hunk 1]
```diff
@@ -70,7 +70,7 @@ class AnchorGenerator(keras.layers.Layer):
     Input shape: an image with shape `[H, W, C]`
     Output: a dictionary with integer keys corresponding to each level of the feature
         pyramid.  The size of the anchors at each level will be
-        `(H/strides * W/strides * K, 4)`.
+        `(H/strides * W/strides * anchors_per_location, 4)`.
```
Comment (tanzhenyu) [inline keras_cv/layers/object_detection/anchor_generator.py:73]: maybe also comment under this line:
`anchors_per_location=len(scales) * len(aspect_ratios)`
?

Comment (LukeWood) [reply keras_cv/layers/object_detection/anchor_generator.py]: Updated - thanks!

**Review by tanzhenyu — APPROVED** [review]

**Pull request merged**